### ISSUE TITLE
Exacerbate by excessive sweating

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10637]
**Preferred term label:**

**Synonyms**
Aggravated by sweating

**Definition (free text, please give PubMed ID)**
PMID: 26945535 High temperatures, high humidity, excessive sweating, pregnancy and delivery, surgery, exposure to UV rays and mechanical irritation are known to be exacerbating factors for DD. 

PMID: 30137841 Wounds in the intertriginous areas can become malodorous, which can be very distressing for the patient. Sun exposure, friction, heat, and sweat can exacerbate the disease.

**Parent term (use hpo.jax.org/app)**
Aggravated by HP:0025285

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:218 DARIER DISEASE

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f 
Adding new term: 
- ID: HP:6001077
- label: Exacerbated by excessive sweating
- parents: [Aggravated by (HP:0025285)](https://hpo.jax.org/app/browse/term/HP:0025285)
- pmid: PMID:26945535
- def: Applied to a sign or symptom that is made worse by excessive sweating.
- comment: High temperatures, high humidity, excessive sweating can exacerbate cutaneous wounds.
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10637
- synonyms:
